### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by carver** [ethereum/py-evm#1941]
### What is wrong?

Windows is not supported. Even if we never get trinity supported, just being able to run evm stuff on Windows would be nice.

### How can it be fixed

Known issues:
- #395 -- need a pyethash fallback (maybe native python?)
- #781 -- fall back to default loop because can't use `uvloop`? Does trio support windows?

Let's add to this list as we discover more.

Then hunt down anything else. Probably need a Windows machine to develop this at any sort of reasonable speed.

**Comment by carver** [comment]
2012 had a "may fix". I think it's highly unlikely that this is resolved. Closing this issue would definitely require confirmation.

**Comment by cburgdorf** [comment]
@carver yep, I guess this was closed automatically from a commit message :+1:

**Comment by 0xClandestine** [comment]
I was able to install on windows using wsl [https://docs.microsoft.com/en-us/windows/wsl/install](wsl)

**Comment by antazoey** [comment]
windows seems to be working now
